org_fedora_oscap/content_handling.py

```diff
@@ -32,7 +32,7 @@
 try:
     from html.parser import HTMLParser
 except ImportError:
-    from HTMLParser import HTMLParser
+    from html.parser import HTMLParser
 
```

Comment:
The `try` section does the same thing as the `except` section now. That isn't correct.

Reply:
@jan-cerny fixed. Apparently wasn't paying attention.